js/hw4.js

```diff
@@ -0,0 +1,226 @@
+// task1
+let Car;
+
+function makeCar(producer, model, production_year, avg_speed) {
+    let car = {
+        producer: producer,
```

Comment:
You can use a shorter version:
```
let car = {
     producer,
     model,
     production_year,
     avg_speed,

...
}
```

Check default names in ES6 ^_^

Reply:
Fixed, here and below - it's what I told about at dailies that I don't use all js special possibilities.